reference-adapter/pspcl-ifix-adapter/src/main/java/org/egov/entity/PspclPaymentDetail.java

```diff
@@ -0,0 +1,37 @@
+package org.egov.entity;
+
+import com.vladmihalcea.hibernate.type.json.JsonBinaryType;
+import lombok.Data;
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Data
+@Entity
+@Table(name = "pspcl_payment_detail")
```

Comment:
Again, would it work if we want to keep db connection modular? I think the db references should only stay in files of the repository package.